specs/integrations/synthetics.md

```diff
@@ -0,0 +1,42 @@
+## Synthetics Integration
+
+Synthetic monitors play a crucial role in periodically checking the status of your services and applications on a global scale. General documentation about synthetic monitors can be found in
+[Synthetics getting started page](https://www.elastic.co/guide/en/observability/current/synthetics-get-started.html).
+
+This integration goes in to more detail about how the sythetics monitors would
+be correlated with the APM traces. Synthetics traces can be categorized in to two
+main types
+  1. HTTP checks - These have one-one mapping with APM transactions
+  2. Browser checks - These have a one-to-many mapping with APM transactions
+
+### Correlation
+
+The Synthetics agent takes the responsibility of creating the [`traceparent`](../agents/tracing-distributed-tracing.md#trace_id-parent_id-and-traceparent) header for each outgoing network request associated with a test during every monitor execution.
+
+- `trace.id` and `parent.id`
+  - outgoing requests that are being explicity traced by the synthetics agent
+     will have the `parent.id` and `trace.id` as part of the trace context.
+  - must be unique for each step for a browser monitor
+  - must be unique for a http monitor
+- `sampled` Flag
+  - used to control the sampling decision for all the downstream services.
+  - 100% sampling when tracing is enabled
+
+These correlation values would be applicable even if the downstream services are traced by
+OpenTelemetry(OTEL)-based agents.
+
+### Identifying Synthetics trace
+
+Synthetics monitor executions creates `rootless traces` as these traces are not
+reported to the APM server. To overcome this limitation on the APM UI, we need
+to identify the synthetics traces and explicity link them  to the Synthetics
+waterfall view. 
+
+- `http.headers.user-agent`:
+  - Contains `Elastic/Synthetics` for all outgoing requests from Synthetis based monitors.
+
+When a trace is confirmed to be originated from Synthetics-based monitors, the
+Trace Explorer view can be linked back to the Synthetics waterfall view.
+
+- `/app/synthetics/link-to/<trace.id>`
```

Comment:
Don't we need both the trace id and the span id?

Reply:
To link back to a particular step for a monitor, we just need the `trace-id`. Synthetics UI does not have a way to link back to a particular span which might be a network request inside a particular step. 

Synthetics URL's take this form right now
```
'/monitor/:monitorId/test-run/:checkGroupId/step/:stepIndex'
```

If we can add the `trace.id` to the Synthetics documents for each step, we can reconstruct the URL's. 

I had removed those details in this issue, Let me add those back.

Reply:
I'd still include the span id in the URL, even if at this point, we can't highlight a span.
But in the future, we could easily extend it to include another path parameter or just an anchor, such as `/monitor/:monitorId/test-run/:checkGroupId/step/:stepIndex#:spanId`.

Reply:
Makes sense though, We could use it to open Network details for the specific waterfall span in the future. Will include this feedback in the doc.

Reply:
Done - https://github.com/elastic/apm/pull/825/commits/7d818cdedef4c97e7286e130f3bc4ccd0e087ef3